Common/Data/AffixRegistry.cs

```diff
@@ -36,15 +36,13 @@ public virtual void Unload() { }
 	/// <returns><see cref="Models.ItemAffixData"/> instance of the given type.</returns>
 	public static ItemAffixData TryGetAffixData(Type type)
 	{
-		try
+		if (ItemAffixData.TryGetValue(type, out ItemAffixData value))
 		{
-			return ItemAffixData[type];
-		}
-		catch (KeyNotFoundException exception)
-		{
-			Console.WriteLine($"ItemAffixData with type {type.Name} not found.\n\n{exception}");
-			return null;
+			return value;
 		}
+
+		Console.WriteLine($"ItemAffixData with type {type.Name} not found.");
```

Comment:
Stdout is not to be written to directly.
```suggestion
		PoTMod.Instance.Logger.Error($"ItemAffixData with type {type.Name} not found.");
```